[2025-05-08T19:37:31.339+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-08T19:37:31.383+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: google_maps_reviews.run_dbt_models manual__2025-05-08T19:31:40.005078+00:00 [queued]>
[2025-05-08T19:37:31.402+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: google_maps_reviews.run_dbt_models manual__2025-05-08T19:31:40.005078+00:00 [queued]>
[2025-05-08T19:37:31.421+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-05-08T19:37:31.447+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): run_dbt_models> on 2025-05-08 19:31:40.005078+00:00
[2025-05-08T19:37:31.464+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=375) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-08T19:37:31.470+0000] {standard_task_runner.py:72} INFO - Started process 390 to run task
[2025-05-08T19:37:31.469+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'google_maps_reviews', 'run_dbt_models', 'manual__2025-05-08T19:31:40.005078+00:00', '--job-id', '852', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmp0zoyc01g']
[2025-05-08T19:37:31.472+0000] {standard_task_runner.py:105} INFO - Job 852: Subtask run_dbt_models
[2025-05-08T19:37:31.570+0000] {task_command.py:467} INFO - Running <TaskInstance: google_maps_reviews.run_dbt_models manual__2025-05-08T19:31:40.005078+00:00 [running]> on host 6188ee5da286
[2025-05-08T19:37:31.749+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='elkhalfimaryame@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='google_maps_reviews' AIRFLOW_CTX_TASK_ID='run_dbt_models' AIRFLOW_CTX_EXECUTION_DATE='2025-05-08T19:31:40.005078+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-08T19:31:40.005078+00:00'
[2025-05-08T19:37:31.751+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-05-08T19:37:31.753+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-05-08T19:37:31.754+0000] {logging_mixin.py:190} INFO - Current task name:run_dbt_models state:running start_date:2025-05-08 19:37:31.384639+00:00
[2025-05-08T19:37:31.755+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews and current dag run status:running
[2025-05-08T19:37:31.756+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-08T19:37:31.759+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-05-08T19:37:31.762+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/dbt_project/bank_reviews_dbt && dbt run --profiles-dir .']
[2025-05-08T19:37:31.788+0000] {subprocess.py:99} INFO - Output:
[2025-05-08T19:37:37.570+0000] {subprocess.py:106} INFO - [0m19:37:37  Running with dbt=1.9.4
[2025-05-08T19:37:38.263+0000] {subprocess.py:106} INFO - [0m19:37:38  Registered adapter: postgres=1.9.0
[2025-05-08T19:37:39.568+0000] {subprocess.py:106} INFO - [0m19:37:39  Found 5 models, 12 data tests, 433 macros
[2025-05-08T19:37:39.575+0000] {subprocess.py:106} INFO - [0m19:37:39
[2025-05-08T19:37:39.577+0000] {subprocess.py:106} INFO - [0m19:37:39  Concurrency: 1 threads (target='dev')
[2025-05-08T19:37:39.579+0000] {subprocess.py:106} INFO - [0m19:37:39
[2025-05-08T19:37:39.883+0000] {subprocess.py:106} INFO - [0m19:37:39  1 of 5 START sql table model review.dim_bank ................................... [RUN]
[2025-05-08T19:37:40.282+0000] {subprocess.py:106} INFO - [0m19:37:40  1 of 5 OK created sql table model review.dim_bank .............................. [[32mSELECT 269[0m in 0.39s]
[2025-05-08T19:37:40.287+0000] {subprocess.py:106} INFO - [0m19:37:40  2 of 5 START sql table model review.dim_branch ................................. [RUN]
[2025-05-08T19:37:40.469+0000] {subprocess.py:106} INFO - [0m19:37:40  2 of 5 OK created sql table model review.dim_branch ............................ [[32mSELECT 269[0m in 0.18s]
[2025-05-08T19:37:40.476+0000] {subprocess.py:106} INFO - [0m19:37:40  3 of 5 START sql table model review.dim_location ............................... [RUN]
[2025-05-08T19:37:40.655+0000] {subprocess.py:106} INFO - [0m19:37:40  3 of 5 OK created sql table model review.dim_location .......................... [[32mSELECT 742[0m in 0.17s]
[2025-05-08T19:37:40.661+0000] {subprocess.py:106} INFO - [0m19:37:40  4 of 5 START sql table model review.dim_sentiment .............................. [RUN]
[2025-05-08T19:37:40.851+0000] {subprocess.py:106} INFO - [0m19:37:40  4 of 5 OK created sql table model review.dim_sentiment ......................... [[32mSELECT 3[0m in 0.19s]
[2025-05-08T19:37:40.859+0000] {subprocess.py:106} INFO - [0m19:37:40  5 of 5 START sql table model review.fact_reviews ............................... [RUN]
[2025-05-08T19:37:41.065+0000] {subprocess.py:106} INFO - [0m19:37:41  5 of 5 OK created sql table model review.fact_reviews .......................... [[32mSELECT 4732[0m in 0.20s]
[2025-05-08T19:37:41.103+0000] {subprocess.py:106} INFO - [0m19:37:41
[2025-05-08T19:37:41.105+0000] {subprocess.py:106} INFO - [0m19:37:41  Finished running 5 table models in 0 hours 0 minutes and 1.52 seconds (1.52s).
[2025-05-08T19:37:41.240+0000] {subprocess.py:106} INFO - [0m19:37:41
[2025-05-08T19:37:41.242+0000] {subprocess.py:106} INFO - [0m19:37:41  [32mCompleted successfully[0m
[2025-05-08T19:37:41.243+0000] {subprocess.py:106} INFO - [0m19:37:41
[2025-05-08T19:37:41.245+0000] {subprocess.py:106} INFO - [0m19:37:41  Done. PASS=5 WARN=0 ERROR=0 SKIP=0 TOTAL=5
[2025-05-08T19:37:42.967+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-05-08T19:37:43.026+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-08T19:37:43.028+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=google_maps_reviews, task_id=run_dbt_models, run_id=manual__2025-05-08T19:31:40.005078+00:00, execution_date=20250508T193140, start_date=20250508T193731, end_date=20250508T193743
[2025-05-08T19:37:43.071+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-05-08T19:37:43.072+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-05-08T19:37:43.073+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews queued_at:2025-05-08 19:31:40.045509+00:00
[2025-05-08T19:37:43.074+0000] {logging_mixin.py:190} INFO - Task hostname:6188ee5da286 operator:BashOperator
[2025-05-08T19:37:43.093+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-08T19:37:43.147+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-08T19:37:43.151+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
